flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/FiniteTestSource.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
```

Comment:
I would move that in the `org.apache.flink.streaming.util` of `flink-test-utils`.